src/views/new evaluation/textarea.js

```diff
@@ -0,0 +1,15 @@
+const Textarea = function (options = {}) {
+return `<div class="content-box-same">
+<h2 class="content-box-head content-box-head-skin">${options.title}</h2>
+<textarea class="content-box-text content-box-text-skin" id="${setKey(options.title)}" rows="2" placeholder='${options.content}'></textarea>
+</div>`;
+};
+    
+    
+    
+    
+const TextareaSection = function (options = {}) {
+    
+const section = options.map(Textarea); // (t) => Textarea(t)
```

Comment:
Just as a side-note, the `section` variable is superfluous and unneeded, since one could very well just do `return options.map(Textarea).join('');` instead.